src/env.ts

```diff
@@ -55,6 +60,9 @@ const logLevelEnvVar = "DD_LOG_LEVEL";
 const logForwardingEnvVar = "DD_FLUSH_TO_LOG";
 const ddTracingEnabledEnvVar = "DD_TRACE_ENABLED";
 const logInjectionEnvVar = "DD_LOGS_INJECTION";
+const subToApiGatewayLogsEnvVar = "DD_SUBSCRIBED_TO_API_GATEWAY_LOGS";
```

Comment:
For these environment variables, I would use `SUBSCRIBE` instead of `SUBSCRIBED`.

"Subscribe" is an imperative verb - we are telling the plugin to subscribe.

"Subscribed" is an adjective, saying that we are, in fact, subscribed. The problem is that we might not actually be subscribed yet. We want to be subscribed, but we don't know if we actually are. So it's a bit misleading to use that here.